tests/test_env.py

```diff
@@ -295,13 +295,15 @@ def test_process_env_install_no_index(tmpdir: py.path.local, monkeypatch, use_sy
     with open(setup_py_path, "w") as setup_file:
         setup_file.write(setup_py_content)
 
-    expected = "Packages this-package-does-not-exist were not found. No indexes were used."
+    expected = re.escape("Packages this-package-does-not-exist were not found. No indexes were used.")
+    other_expected = r"Packages setuptools.*were not found\. No indexes were used\."
 
-    with pytest.raises(env.PackageNotFound, match=re.escape(expected)):
+    with pytest.raises(env.PackageNotFound, match=f"{expected}|{other_expected}"):
```

Comment:
This needs a docstring. Why do we expect one of these error messages?

Reply:
Because we install two things, they fail differently, but order is not deterministic